pkg/asset/machines/master.go

```diff
@@ -648,9 +665,61 @@ func (m *Master) Generate(ctx context.Context, dependencies asset.Parents) error
 			Data:     data,
 		}
 	}
+
+	// This is only used by Two Node OpenShift with Fencing (TNF)
+	// The credentials are rendered into secrets that will be consumed by the
+	// cluster-etcd operator to enable recovery via fencing
+	if pool.Fencing != nil && pool.Fencing.Credentials != nil {
```

Comment:
```suggestion
	if pool.Fencing != nil && len(pool.Fencing.Credentials) > 0 {
```

Looks like we can skip if it's empty?